client/coral-configure/containers/ConfigureStreamContainer.js

```diff
@@ -53,18 +53,20 @@ class ConfigureStreamContainer extends Component {
 
   handleChange (e) {
     if (e.target && e.target.id === 'qboxenable') {
-      this.props.asset.settings.questionBoxEnable = e.target.checked;
+      this.state.settings.questionBoxEnable = e.target.checked;
```

Comment:
One reason is that React expects exclusive access to the state. To avoid any hard to debug bugs in the future and to maintain compatibility with future React versions, we should only mutate the state using `setState`. 

Also we want to treat state as an immutable object, to be able to apply performance optimisations to the component. In this example, you can construct a new settings object using `settings = {...this.state.settings, questonBoxEnable: true};`. Also because we won't need `premodLinksEnable` nor `questionBoxContent `, I would just avoid using the `settings` object and just put `questonBoxEnable` on the state.